container_test.go

```diff
@@ -294,9 +293,8 @@ func TestCycles(t *testing.T) {
 	require.NoError(t, c.Provide(c1))
 	require.NoError(t, c.Provide(c2))
 	err := c.Provide(c3)
-
-	require.Contains(t, err.Error(), "unable to Provide dig.Type3")
-	require.Contains(t, err.Error(), "dig.Type3 -> dig.Type1 -> dig.Type3")
+	require.Contains(t, err.Error(), "unable to Provide [dig.Type3]")
+	require.Contains(t, err.Error(), "func(dig.Type1) dig.Type3 -> func(dig.Type2, dig.Type3) dig.Type1 -> func(dig.Type1) dig.Type3")
```

Comment:
can we add a test somewhere that shows `func(type1, type2) type3, type4 -> func(type4) type1` or something like that?